.github/workflows/nbgl_tests.yml

```diff
@@ -0,0 +1,24 @@
+name: Run NBGL tests
+
+on:
+  push:
+    branches:
+      - master
+      - main
+      - develop
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  run_NBGL_tests:
+    runs-on: ubuntu-latest
```

Comment:
Would it work more simply using something like:

```
jobs:
  run_NBGL_tests:
    uses: LedgerHQ/app-nbgl-tests/.github/workflows/build_and_functional_tests.yml@master
    with:
      golden_run: "Open a PR"
    secrets: inherit
```
:thinking:

Reply:
yes could be interesting if it's working and if the PR can be opened in the remote repository

Reply:
https://github.com/LedgerHQ/ledger-secure-sdk/actions/runs/12199393685
It seems that this option is not working